app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,125 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+ * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+$lightgray:   #D3D3D3;
+$mediumgray:  #989898;
+$darkgray:    #585858;
+
+html {
+  font-family: Arial, sans-serif;
+  text-align: center;
+  background-color: white;
+}
+
+body {
+  width: 960px;
+  background-color: $lightgray;
+  margin: 100px auto;
+  padding: 0 0 25px 0;
```

Comment:
Setting a general override for your CSS would help you avoid needing to do super specific things like this padding.